From f1c6c319441c58f1e9c8182a016bf225460c4886 Mon Sep 17 00:00:00 2001
From: CaledoniaProject <CaledoniaProject@users.noreply.github.com>
Date: Tue, 17 Oct 2023 08:45:59 -0500
Subject: [PATCH] Update stub-pcap.c to support centos 7

On CentOS7 libpcap is installed as libpcap.so.1, adding thisto possible names
---
 src/stub-pcap.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/stub-pcap.c b/src/stub-pcap.c
index d93554f..f8f6cac 100644
--- a/src/stub-pcap.c
+++ b/src/stub-pcap.c
@@ -371,6 +371,7 @@ if (pl->datalink == NULL) pl->func_err=1, pl->datalink = null_##PCAP_DATALINK;
             "libpcap.so",
             "libpcap.A.dylib",
             "libpcap.dylib",
+            "libpcap.so.1",
             "libpcap.so.0.9.5",
             "libpcap.so.0.9.4",
             "libpcap.so.0.8",
-- 
2.51.0

